lang/test/org/partiql/lang/eval/LikePredicateTest.kt

```diff
@@ -629,5 +630,33 @@ class LikePredicateTest : EvaluatorTestBase() {
     }
 
 
+    @Test
+    fun interruptedThreadThrowsInterruptedException() {
+        val sb = StringBuilder()
+        sb.append("foo LIKE '%")
+        repeat(5000) {
+            sb.append("!")
+        }
```

Comment:
nit: Could replace with [repeat](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/repeat.html).

`sb.append("!".repeat(5000))`

Reply:
Actually, `"!"` enables removal of the `StringBuilder` too.  thanks.